spec/formatting.md

```diff
@@ -197,17 +197,31 @@ the following steps are taken:
 
 1. If the _expression_ includes an _operand_, resolve its value.
    If this fails, use a _fallback value_ for the _expression_.
-2. Based on the _function_ starting sigil and _name_,
-   find the appropriate function implementation from the _function registry_.
-   If the registry does not define an implementation for this _name_,
-   emit an Unknown Function error
+2. Resolve the _identifier_ of the _function_ and, based on the starting sigil,
+   find the appropriate function implementation to call.
+   If the implementation cannot find the function, emit an Unknown Function error
```

Comment:
@aphillips Would this change (together with removing the next paragraph) be minimally sufficient to covering all cases? Not supporting _any_ namespaces and not supporting _this_ namespace are pretty much the same at this point.

```suggestion
   If the implementation cannot find the function,
   or if the _identifier_ includes a _namespace_ that the implementation does not support,
   emit an Unknown Function error
```